File: trail1/5의 배수의 합/sum-of-specific-condition-multiple.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        int ssum = 0;
        if (A > B) {
            int temp = B;
            B = A;
            A = temp;
        }
        for (int i=A; i<=B; i++) {
            if (i % 5 == 0) {ssum += i;}
        }
        System.out.println(ssum);
    }
}

File: trail1/5의 배수의 합/sum-of-specific-condition-multiple.cpp
#include <iostream>
using namespace std;
void swap(int &x, int &y) {
    int temp = y;
    y = x;
    x = temp;
}

int main() {
    int A, B;
    cin >> A >> B;
    int ssum = 0;
    if (A > B) {swap(A, B);}
    for (int i=A; i<=B; i++) {
        if (i % 5 == 0) {ssum += i;}
    }
    cout << ssum << endl;
    return 0;
}